### ISSUE TITLE
Reorder accessibility elements in chat list

### ISSUE THREAD

**Issue by cyBerta** [deltachat/deltachat-ios#1538]
we should have a look at telegram, chat list entries should be read out in the following order: 

- 1. group title
- 2. unread messages count if available "x unread messages"
- 3. "message from <sender>": cell subtitle
- 4. read out time

currently the order is: group title, time, subtitle, unread messages cout if available
